AUTHORS

```diff
@@ -233,7 +233,6 @@ Prasoon Shukla <prasoon92.iitr@gmail.com>
 Stefen Yin <zqyin@ucdavis.edu>
 Thomas Hisch <t.hisch@gmail.com>
 Madeleine Ball <mpball@gmail.com>
-Case Van Horsen <casevh@gmail.com>
```

Comment:
I have added a new commit as well to update `.mailmap` as done accordingly in [Line 255](https://github.com/sympy/sympy/blob/master/.mailmap#L255) and [Line 256](https://github.com/sympy/sympy/blob/master/.mailmap#L256) in the original file